src/System.IO/tests/MemoryStream/MemoryStream.GetBufferTests.cs

```diff
@@ -31,6 +31,15 @@ public void MemoryStream_GetBuffer_Exposable()
             Assert.Equal(500, buffer.Length);
         }
 
+        [ActiveIssue("https://github.com/dotnet/coreclr/pull/23700")]
```

Comment:
You'll remove active issue before committing?

Reply:
Yes, that's why I haven't merged yet :-)